src/index.js

```diff
@@ -156,25 +157,26 @@ export const validateModulesOption = (modulesOpt = "commonjs") => {
   return modulesOpt;
 };
 
-export const validateWhitelistOption = (whitelistOpt = []) => {
-  if (!Array.isArray(whitelistOpt)) {
-    throw new Error(`The 'whitelist' option must be an Array<string> of plugins
-      {
-        "presets": [
-          ["env", {
-            "targets": {
-              "chrome": 50
-            },
-            "whitelist": ["transform-es2015-arrow-functions"]
-          }]
-        ]
-      }
-      was passed "${whitelistOpt}" instead
-    `);
+function validatePluginsOption(opts = [], type) {
+  if (!Array.isArray(opts)) {
+    throw new Error(`The '${type}' option must be an Array<string> of plugins`);
   }
 
-  return whitelistOpt;
-};
+  for (let opt of opts) {
```

Comment:
Actually, this whole thing could be:

```js
difference([
  ...Object.keys(pluginFeatures),
  ...Object.values(MODULE_TRANSFORMATIONS)
], opts);
```

Which would have the added value of being able to call out multiple errors at once rather than bailing at the first one.

https://lodash.com/docs/4.17.2#difference